pkg/config/config_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 )
 
-func TestRead(t *testing.T) {
+func TestRead2(t *testing.T) {
```

Comment:
Why change the name? It should be the same as the name of the function that it tests

```suggestion
func TestRead(t *testing.T) {
```

Reply:
That was an accident I think. I used code generation to create another test, but removed it again, but not cleanly.